src/test/java/org/sonar/plugins/checkstyle/internal/ChecksTest.java

```diff
@@ -189,7 +189,7 @@ private static void validateSonarRuleProperties(Class<?> module, Set<Node> param
             final String paramKey = paramKeyNode.getTextContent();
 
             Assert.assertFalse(moduleName
-                    + " requires a valid key for unknown parameter in sonar rules"
+                            + " requires a valid key for unknown parameter in sonar rules"
```

Comment:
revert such formatting or move to separate commit.
same for all other like this